code/game/objects/effects/overlays.dm

```diff
@@ -559,14 +561,14 @@
 	icon_state = "impact_bullet"
 	duration = 5
 
-/obj/effect/overlay/temp/impact_effect/New(loc, atom/target, obj/item/projectile/P)
+/obj/effect/overlay/temp/impact_effect/Initialize(mapload, atom/target, obj/item/projectile/P)
 	if(target == P.original) //the projectile hit the target originally clicked
 		pixel_x = P.p_x + target.pixel_x - 16 + rand(-4,4)
 		pixel_y = P.p_y + target.pixel_y - 16 + rand(-4,4)
 	else
 		pixel_x = target.pixel_x + rand(-4,4)
 		pixel_y = target.pixel_y + rand(-4,4)
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?